commit bb3852f0406b7452ccebd837183dcd2817060b62
Author: stephen waite
Date:   2024-10-03

    fix 02 auto sftp


diff --git a/src/Billing/BillingProcessor/X12RemoteTracker.php b/src/Billing/BillingProcessor/X12RemoteTracker.php
index 20cc693bc7..8d71509733 100644
--- a/src/Billing/BillingProcessor/X12RemoteTracker.php
+++ b/src/Billing/BillingProcessor/X12RemoteTracker.php
@@ -76,6 +76,14 @@ class X12RemoteTracker extends BaseService
                 $claim_file = $GLOBALS['OE_SITE_DIR'] . "/documents/edi/" . $x12_remote['x12_filename'];
             }
 
+            if ($x12_remote['x12_sftp_host'] == 'moveit.bcbsvt.com') {
+                $moveitFilename = '007111' . rand(20, 99) . '.x12';
+                rename($claim_file, $GLOBALS['OE_SITE_DIR'] . "/documents/edi/" . $moveitFilename);
+                $claim_file = $GLOBALS['OE_SITE_DIR'] . "/documents/edi/" . $moveitFilename;
+                $x12_remote['x12_filename'] = $moveitFilename;
+            }
+
+
             $claim_file_contents = file_get_contents($claim_file);
             if (false === $claim_file_contents) {
                 $x12_remote['status'] = self::STATUS_CLAIM_FILE_ERROR;
diff --git a/src/Billing/Claim.php b/src/Billing/Claim.php
index 88ce1c333e..fb4f4b7ad3 100644
--- a/src/Billing/Claim.php
+++ b/src/Billing/Claim.php
@@ -748,7 +748,7 @@ class Claim
 
     public function payToFacilityStreet()
     {
-        return $this->x12Clean(trim($this->billing_facility['mail_street']));
+        return $this->x12Clean(trim($this->billing_facility['mail_street'] ?? ''));
     }
 
     public function payToFacilityStreet2()
diff --git a/src/Billing/X125010837P.php b/src/Billing/X125010837P.php
index 7a3747c0e3..57fe12eb82 100644
--- a/src/Billing/X125010837P.php
+++ b/src/Billing/X125010837P.php
@@ -1585,7 +1585,7 @@ class X125010837P
                 // for esp
                 if (
                     $_SESSION['site_id'] == '1500'
-                    && in_array($claim->payerID(), ['87726', '39026', '25169', 'TREST', 'PAMCD'])
+                    && in_array($claim->payerID(), ['87726', '39026', 'TREST', 'PAMCD'])
                 ) {
                     continue;
                 }
