commit 907594e8a0a43909cdfca9642ab25d768b564b13
Author: stephen waite
Date:   2023-05-25

    update dls label


diff --git a/interface/forms/newpatient/common.php b/interface/forms/newpatient/common.php
index ed0fa74841..e6f7780538 100644
--- a/interface/forms/newpatient/common.php
+++ b/interface/forms/newpatient/common.php
@@ -523,7 +523,7 @@ $ires = sqlStatement("SELECT id, type, title, begdate FROM lists WHERE " .
                         </div>
                         <div class="col-sm <?php echo ($GLOBALS['gbl_visit_onset_date'] == 1) ?: 'd-none'; ?>">
                             <div class="form-group">
-                                <label for='form_onset_date' class="text-right"><?php echo xlt('Onset/hosp. date:'); ?> &nbsp;<i id='onset-tooltip' class="fa fa-info-circle text-primary" aria-hidden="true"></i></label>
+                                <label for='form_onset_date' class="text-right"><?php echo text($date_label); ?> &nbsp;<i id='onset-tooltip' class="fa fa-info-circle text-primary" aria-hidden="true"></i></label>
                                 <input type='text' class='form-control datepicker' name='form_onset_date' id='form_onset_date' value='<?php echo $viewmode && $result['onset_date'] !== '0000-00-00 00:00:00' ? attr(oeFormatDateTime($result['onset_date'])) : ''; ?>' title='<?php echo xla('Date of onset or hospitalization'); ?>' />
                             </div>
                         </div>
