commit bf2848ba3e571d9e08b38155dfda1ed90f175503
Author: stephen waite
Date:   2024-10-02

    fix billing report topatient toencounter


diff --git a/interface/billing/billing_report.php b/interface/billing/billing_report.php
index 851f7b18e6..7ab37d6149 100644
--- a/interface/billing/billing_report.php
+++ b/interface/billing/billing_report.php
@@ -347,9 +347,12 @@ $partners = $x->_utility_array($x->x12_partner_factory());
             top.restoreSession();
             encurl = 'patient_file/encounter/encounter_top.php?set_encounter=' + encodeURIComponent(enc) +
                 '&pid=' + encodeURIComponent(pid);
+            paturl = 'patient_file/summary/demographics.php?pid=' + encodeURIComponent(pid);
+            parent.clearPatient();
             parent.left_nav.setPatient(pname, pid, pubpid, '', dobstr);
             parent.left_nav.setEncounter(datestr, enc, 'enc');
             parent.left_nav.loadFrame('enc2', 'enc', encurl);
+            parent.left_nav.loadFrame('dem1', 'pat', paturl);
         }
 
         // Process a click to go to an patient.
@@ -357,7 +360,9 @@ $partners = $x->_utility_array($x->x12_partner_factory());
             top.restoreSession();
             paturl = 'patient_file/summary/demographics_full.php?pid=' + encodeURIComponent(pid);
             parent.left_nav.setPatient(pname, pid, pubpid, '', dobstr);
+            parent.left_nav.loadFrame('ens1', 'enc', 'patient_file/history/encounters.php?pid=' + encodeURIComponent(pid));
             parent.left_nav.loadFrame('dem1', 'pat', paturl);
+            parent.activateTabByName('pat', true);
         }
 
         function popMBO(pid, enc, mboid) {
